.pre-commit-config.yaml

```diff
@@ -28,7 +28,7 @@ repos:
     hooks:
     -   id: python-use-type-annotations
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.11.2
+    rev: v1.12.1
```

Comment:
this one exposes new linting violations

Reply:
Removed false negatives are easy to fix, I'm a little less certain about the new false positives.